### ISSUE TITLE
Spring Boot 3.3.1 new warning about `DelegatingWsConfiguration is not eligible for getting processed by all BeanPostProcessors`

### ISSUE THREAD

**Issue by CC007** [spring-projects/spring-ws#1425]
This is a continuation of #1391, since the problem still persists. 

Aside from that, I also noticed that this same error also occurs when extending the `WsConfigAdapter`, and I'm not the only one. See [this StackOverflow question](https://stackoverflow.com/questions/73360644/spring-configuration-bean-not-eligible-for-getting-processed-by-beanpostprocesso)

**Comment by flaviobarisoneMD** [comment]
Have this warning any consequences on spring-ws module features?

**Comment by CC007** [comment]
Yes it could have consequences. Spring doesn't log this without reason: https://www.baeldung.com/spring-not-eligible-for-auto-proxying

**Comment by CC007** [comment]
I don't think users can fix this after the fact. It's something that would need to be fixed in the library. The library may or may not rely on this post-processing. This needs to be evaluated, to see if this causes any issues in the functionality of the library.

**Comment by snicoll** [comment]
I've reopened #1391.
